code/Attack.ipynb

```diff
@@ -0,0 +1,1466 @@
+{
```

Comment:
<p>Include descriptions for each of the artifacts. We don't know which Transforer model(s) is being used. Which test set is being used, and so on. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/robustness-foundation-models/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>